Assignment2/crisfernandez.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix:<http://example.org/assignment2/#>.
+@prefix pred:<http://example.org/assignment2/predicates#>.
+@prefix xsd:<http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 pred:includes :Sensor029;pred:includes :Computer101.
+
+:Sensor029 pred:hasMeasurement :Measurement8401.
+
+:Measurement8401 pred:hasTemperature "29"^^xsd:integer;
+pred:atTime "2010-06-12T12"^^xsd:dateTime.
```

Comment:
Not a perfect dateTime, as the content in the string will not parse correctly. But I accept it so as not to have too many pending pull requests.